lib/content-services/src/lib/tag/services/tag.service.spec.ts

```diff
@@ -15,13 +15,20 @@
  * limitations under the License.
  */
 
-import { setupTestBed } from '@alfresco/adf-core';
+import { LogService, setupTestBed } from '@alfresco/adf-core';
 import { TagService } from './tag.service';
-import { TestBed } from '@angular/core/testing';
+import { fakeAsync, TestBed, tick } from '@angular/core/testing';
 import { ContentTestingModule } from '../../testing/content.testing.module';
 import { TranslateModule } from '@ngx-translate/core';
 import { throwError } from 'rxjs';
-import { TagEntry } from '@alfresco/js-api';
+import {
+    RequestQuery,
+    RequestSortDefinitionInner,
+    ResultSetPaging,
+    SearchApi, SearchRequest,
```

Comment:
can we add SearchRequest import in next line.

Reply:
done